hardware_interface/test/test_component_parser.cpp

```diff
@@ -20,6 +20,10 @@
 #include "ros2_control_test_assets/components_urdfs.hpp"
 #include "ros2_control_test_assets/descriptions.hpp"
 
+#ifndef M_PI
+#define M_PI 3.1415926535897932384626433832795
+#endif
```

Comment:
And probably here something line `#ifdef WIN_64`
```suggestion
#define _USE_MATH_DEFINES
#include <cmath>
```

Reply:
hello.
It seems that `<cmath>` should be included before `gmock/gmock.h`.
In my environment, including `cmath` after `gmock/gmock.h` didn't work but before it worked.